Bugsnag/Assets/Bugsnag/Editor/BuildPreprocessor.cs

```diff
@@ -9,16 +9,22 @@ public class BuildPreprocessor : IPreprocessBuildWithReport
 
     public void OnPreprocessBuild(BuildReport report)
     {
+        //This will be enabled once IL2CPP source mapping is supported
+        #if UNITY_6000_0_OR_NEWER
```

Comment:
Small suggestion: perhaps it's worth abstracting a `PlayerSettingsCompat` class that contains this conditioning for any access to PlayerSettings? Then we're less likely to forget to use it in future and the code isn't littered with ifdefs.

Reply:
I like this idea, will do it 👍